lib/ftp/bifrost.ex

```diff
@@ -128,11 +128,12 @@ defmodule Ftp.Bifrost do
       ) do
     case {username, password} do
       {^expected_username, ^expected_password} ->
-        Ftp.EventDispatcher.dispatch(:e_login_successful)
+        Ftp.EventDispatcher.dispatch(:e_login_successful, state)
+        IO.puts("This is state: #{inspect state}")
```

Comment:
IO.puts? I presume it is some debugging residue...

Reply:
Yep, well spotted! I have removed it now.